src/applications/check-in/travel-claim/pages/complete/TravelClaimSuccessAlert.jsx

```diff
@@ -12,16 +17,29 @@ const TravelClaimSuccessAlert = () => {
         status="success"
         uswds
       >
-        <h2 slot="headline">{t('claim-submitted', { count: 1 })}</h2>
+        <h2 slot="headline">{t('claim-submitted')}</h2>
         <p
           className="vads-u-margin-top--0"
           data-testid="travel-pay--claim--submitted"
         >
-          {`${t('this-claim-is-for-your-appointment', {
-            date: format(new Date(), 'MMMM dd, yyyy'),
-            claims: 1,
-            appointments: 1,
-          })} ${t('well-send-you-a-text-to-let-you-know', { count: 1 })}`}
+          {`${t('this-claim-is-for-your-appointment')} ${t('appointment-on', {
```

Comment:
I thought this should read `...for your in person appointment` but I could be confused.

Reply:
no, you are correct, I missed that update in the Figma. I'll get it re-worked.